mod/tidypics/actions/photos/album/save.php

```diff
@@ -49,10 +49,10 @@
 $album->access_id = $access_id;
 $album->title = $title;
 $album->title2 = $title2;
-$album->title3 = gc_implode_translation($title,$title2);
+$album->title = gc_implode_translation($title,$title2);
```

Comment:
should this move up instead of setting this twice in the same block? Description is doing the same below